admin/app/views/workarea/admin/users/index.json.jbuilder

```diff
@@ -2,3 +2,11 @@ json.results @results do |user|
   json.label user.name
   json.value user.id
 end
+
+json.mentions @results do |user|
```

Comment:
Any reason we can't combine the extra fields into the results above so it's one data set?

Reply:
A few reasons I did it this way:

- Create a dataset that serves a different module than Select2, since they require different data
- Make the module more direct, bypassing the need to munge the returned data into something usable for Tribute
- Be a more clear in intention of each data set

The `tributeAreas` module got a bit heftier than I originally anticipated because Tribute has no hooks for determining when a user changed their mind about a mention. A totally reasonable use-case would be if someone started typing a name, saw the name in the list, and accidentally clicked the wrong name. When they delete their mistake no events are triggered in tribute to announce that this has happened, so I had to write that into the `tributeAreas` module itself.

I felt that getting the data in the format that tribute needed was a low cost and high yield kind of move to save me from having to write another method in the module with high cognitive overhead.